### PR TITLE
[TW-1228] Doc you can’t get Excel to wrap CSV numbers in quotes

### PR DESCRIPTION
**Pull request by nerminamiller-postman**
Updates admonition about formatting numbers with leading zeroes

### CODE REVIEW

**Review by mark-anthony-postman — APPROVED** [review]
LGTM!

File: src/pages/docs/collections/running-collections/working-with-data-files.md [hunk 1]
```diff
@@ -23,6 +23,14 @@ contextual_links:
 
 You can use data files to pass Postman sets of values to use in a collection run. By selecting a JSON or CSV data file in the Collection Runner, you can test your requests with multiple values as part of a single run.
 
+Before exporting your file, take into account the following considerations.
+
+* CSV values with preceding zeroes (for example, `000000345`) are not correctly parsed by Postman unless wrapped in double quotation marks (for example, `"000000345"`). Also, common spreadsheet programs (Microsoft Excel and Google Sheets) do not wrap values in quotes unless the value contains a comma or a double quotation mark.
+
+* To treat numbers with leading zeroes as strings in Postman, reopen your exported CSV file in a text editor (or create a script) to append double quotes to the numbers.
```
Comment (nerminamiller-postman) [inline src/pages/docs/collections/running-collections/working-with-data-files.md:30]:
```suggestion
* To treat numbers with leading zeroes as strings in Postman, reopen your exported CSV file in a text editor (or create a script) to wrap the number in double quotes.
```

File: src/pages/docs/collections/running-collections/working-with-data-files.md [hunk 2]
```diff
@@ -23,6 +23,14 @@ contextual_links:
 
 You can use data files to pass Postman sets of values to use in a collection run. By selecting a JSON or CSV data file in the Collection Runner, you can test your requests with multiple values as part of a single run.
 
+Before exporting your file, take into account the following considerations.
+
+* CSV values with preceding zeroes (for example, `000000345`) are not correctly parsed by Postman unless wrapped in double quotation marks (for example, `"000000345"`). Also, common spreadsheet programs (Microsoft Excel and Google Sheets) do not wrap values in quotes unless the value contains a comma or a double quotation mark.
+
+* To treat numbers with leading zeroes as strings in Postman, reopen your exported CSV file in a text editor (or create a script) to wrap the number in double quotes.
```
Comment (bryancross) [inline src/pages/docs/collections/running-collections/working-with-data-files.md:30]: This is great!  Boo excel for dumbing down their export capabilities

File: src/pages/docs/collections/running-collections/working-with-data-files.md [hunk 3]
```diff
@@ -23,6 +23,14 @@ contextual_links:
 
 You can use data files to pass Postman sets of values to use in a collection run. By selecting a JSON or CSV data file in the Collection Runner, you can test your requests with multiple values as part of a single run.
 
+Before exporting your file, take into account the following considerations.
+
+* CSV values with preceding zeroes (for example, `000000345`) are not correctly parsed by Postman unless wrapped in double quotation marks (for example, `"000000345"`). Also, common spreadsheet programs (Microsoft Excel and Google Sheets) do not wrap values in quotes unless the value contains a comma or a double quotation mark.
```
Comment (bryancross) [inline src/pages/docs/collections/running-collections/working-with-data-files.md:28]: This is absolutely honest, but having said that might suggest removing 'not correctly' and perhaps replacing it with something like 'Leading zeros are stripped from CVS fields by Postman...'  Everything Postman does is correct 👍

File: src/pages/docs/collections/running-collections/working-with-data-files.md [hunk 4]
```diff
@@ -23,6 +23,14 @@ contextual_links:
...
+
+* To treat numbers with leading zeroes as strings in Postman, reopen your exported CSV file in a text editor (or create a script) to wrap the number in double quotes.
+
+* You do not need to add double quotes to simple integers. You do, however, need to format large numbers as text in your spreadsheet program, so they are not truncated during export.
```
Comment (bryancross) [inline src/pages/docs/collections/running-collections/working-with-data-files.md:32]: What are the ranges for 'large numbers'?

File: src/pages/docs/collections/running-collections/working-with-data-files.md [hunk 5]
```diff
@@ -71,14 +79,14 @@ You may encounter errors when Postman attempts to read your data file during a c
 
 * Ensure your data file is formatted as either CSV or JSON.
 
-* If you want to keep large numbers or numbers with leading zeros, specify them as a string in the CSV file by wrapping the number in double quotes as follows:
+* To ensure the correct encoding of your data file, open the file in a text editor and save it in a different format.
```
Comment (bryancross) [inline src/pages/docs/collections/running-collections/working-with-data-files.md:82]: not sure what this means.  If I have a CSV file, do I have to save it as a text file or ... ?

File: src/pages/docs/collections/running-collections/working-with-data-files.md (hunk 3 above)
Comment (nerminamiller-postman) [reply src/pages/docs/collections/running-collections/working-with-data-files.md]: Updated!

File: src/pages/docs/collections/running-collections/working-with-data-files.md (hunk 4 above)
Comment (nerminamiller-postman) [reply src/pages/docs/collections/running-collections/working-with-data-files.md]: Updated!

File: src/pages/docs/collections/running-collections/working-with-data-files.md (hunk 5 above)
Comment (nerminamiller-postman) [reply src/pages/docs/collections/running-collections/working-with-data-files.md]: Removed this sentence as it wasn’t clear to me either. It was there before but it doesn’t make sense to change the format.

**Review by bryancross — APPROVED** [review]

**Pull request merged**